Solutions/Jamf Protect/Analytic Rules/JamfProtectAlerts.yaml

```diff
@@ -13,26 +13,16 @@ suppressionEnabled: false
-  jamfprotect_CL
-  | where topicType_s == "alert"
-    and input_eventType_s <> "GPUnifiedLogEvent"
-    and isnotempty(input_match_severity_d)
-  | extend severity = case(input_match_severity_d == 0, "Informational", input_match_severity_d == 1, "Low", input_match_severity_d == 2, "Medium", input_match_severity_d == 3, "High", "Informational")
-  | extend Match_facts = parse_json(input_match_facts_s)
-  | extend Analytic_description =  Match_facts[0].human
-  | extend Analytic_name =  Match_facts[0].name
-  | extend AlertURL = strcat("https://", context_identity_claims_hd_s, ".jamfcloud.com/Alerts/", input_match_uuid_g)
-  | extend Input_Match = parse_json(input_match_actions_s)
-  | extend sha256hex = tostring(parse_json(input_related_binaries_s)[0].sha256hex)
-  | extend algorithm = "SHA256"
-  | extend Matched_Users = tostring(parse_json(input_related_users_s)[0].name)
-  | extend Host_IPs = tostring(parse_json(input_host_ips_s)[0])
-  | extend Tags = tostring(Match_facts[0].tags)
-  | extend Tactics = case(Tags has "Execution", "Execution", Tags has "Visibility", "Visibility", Tags has "Persistence", "Persistence", Tags has "LateralMovement", "Lateral Movement", Tags has "CredentialAccess", "Credential Acccess", Tags has "DefenseEvasion", "Defense Evasion", Tags has "PrivilegeEscalation", "Privilege Escalation", Tags has "Impact", "Impact", Tags has "CommandAndControl", "Command and Control", Tags has "Discovery", "Discovery", Tags has "InitialAccess", "Initial Access", "")
-  | extend Techniques = case(Tags has "T1003", "T1003", Tags has "T1036", "T1036", Tags has "T1204", "T1204", Tags has "T1068", "T1068", Tags has "T1485", "T1485", Tags has "T1489", "T1489", Tags has "T1219", "T1219", Tags has "T1219", "T1219", Tags has "T1059", "T1059", Tags has "T1557", "T1557",Tags has "T1090", "T1090", Tags has "T1547", "T1547", Tags has "T1204", "T1204", Tags has "T1068", "T1068", Tags has "T1053", "T1053", "")
-  | extend JamfPro = case(Input_Match has "SmartGroup", "Workflow with Jamf Pro", Input_Match has "Prevented", "No workflow, Prevented by Protect", "No workflow")
-  | extend ProviderName = "Jamf"
-  | extend ProductName = "Jamf Protect"
+  JamfProtect
+  | where EventProduct == "Jamf Protect - Alerts"
+      and isnotempty(EventSeverity)
+  | extend
+      algorithm = "SHA256",
+      Host_IPs = tostring(parse_json(DvcIpAddr)[0]),
+      Tags = tostring(Match_facts[0].tags),
+      Tactics = case(Match_tags has "Execution", "Execution", Match_tags has "Visibility", "Visibility", Match_tags has "Persistence", "Persistence", Match_tags has "LateralMovement", "Lateral Movement", Match_tags has "CredentialAccess", "Credential Acccess", Match_tags has "DefenseEvasion", "Defense Evasion", Match_tags has "PrivilegeEscalation", "Privilege Escalation", Match_tags has "Impact", "Impact", Match_tags has "CommandAndControl", "Command and Control", Match_tags has "Discovery", "Discovery", Match_tags has "InitialAccess", "Initial Access", ""),
```

Comment:
Hi @txhaflaire, I confirmed with the team regarding the dynamically populating the tactics and techniques, that's fine. 

But there is small issue, Tactics should not have spaces. For example, here "Lateral Movement", "Credential Acccess" etc. have spaces, please remove them.
You can also refer to [Query Style Guide](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide)

Reply:
@v-rbajaj Thanks for confirming.

Pushed the change, alongside generation a new solution package.